Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppDeploy/DeployIISWebApp.ps1

```diff
@@ -70,7 +72,8 @@ function Run-RemoteDeployment
         [string]$adminPassword,
         [string]$winrmProtocol,
         [string]$testCertificate,
-        [string]$deployInParallel
+        [string]$deployInParallel,
+        [string]$webDeployPackage
```

Comment:
No Invoke-RemoteDeployment is agnostic to what we are deploying it just takes script and runs on the target machines. So for customer we are giving context here saying deploying website or deploying pkg etc etc